.vscode/settings.json

```diff
@@ -27,5 +27,7 @@
   ],
   "i18n-ally.localesPaths": ["client/src/assets/i18n"],
   "i18n-ally.enabledFrameworks": ["ngx-translate"],
-  "i18n-ally.keystyle": "nested"
+  "i18n-ally.keystyle": "nested",
+  "appService.defaultWebAppToDeploy": "/subscriptions/b2d243bd-7fab-4a8a-8261-a725ee0e3b47/resourceGroups/510Global/providers/Microsoft.Web/sites/CPDW-Development",
```

Comment:
@RubenGeo  I don't think these details should be committed into the repository..

But how I understand it now is that the deployment is triggered manually via the VS-Code extension?

That'll be fine for now, but I think we can set it up to deploy after each merge-of-/server-code-on-main, isnt' it?
But we can so that in a later PR...